podvm/Dockerfile.podvm

```diff
@@ -33,22 +33,23 @@ RUN if [ -n "${CAA_SRC}" ]; then \
 # Installs add-ons for foreign target, if required
 RUN ./cloud-api-adaptor/podvm/hack/cross-build-extras.sh
 
-# Default to Ubuntu Focal amd64 release image. These variables can be overriden as needed
+# Defaults to Ubuntu Focal amd64 release image. These variables can be overriden as needed
 ARG UBUNTU_IMAGE_URL=https://cloud-images.ubuntu.com/releases/focal/release-20230107/ubuntu-20.04-server-cloudimg-amd64.img
 ARG UBUNTU_IMAGE_CHECKSUM=3895e38566e5c2c019f5c6f825ab7570ee34dac6b9142fab0c7e5a78084c4280
 
 ENV UBUNTU_IMAGE_URL ${UBUNTU_IMAGE_URL}
 ENV UBUNTU_IMAGE_CHECKSUM ${UBUNTU_IMAGE_CHECKSUM}
 
+# Copy the binaries to podvm/files folder
+COPY --from=podvm_binaries /podvm-binaries.tar.gz /src/cloud-api-adaptor/podvm/files
+RUN tar xvf /src/cloud-api-adaptor/podvm/files/podvm-binaries.tar.gz -C /src/cloud-api-adaptor/podvm/files
 
-# Building binaries separately allows for reuse in case of error
-# with image building step OTOH it may increase layers size
-#RUN cd cloud-api-adaptor/podvm && \
-#     make binaries
```

Comment:
I'd prefer to leave it but not a deal but fine with removing as well

Reply:
Removed it to avoid any confusion. But I can add commented out code for anyone wanting to rebuilding the binaries when creating the image